OPHD/States/PlanetSelectState.cpp

```diff
@@ -115,9 +115,9 @@ State* PlanetSelectState::update()
 		mPlanets[i]->update();
 	}
 
-	renderer.drawText(mFontBold, "Mercury Type", mPlanets[0]->position() + NAS2D::Vector{64 - (mFontBold.width("Mercury Type") / 2), -mFontBold.height() - 10}, NAS2D::Color::White);
-	renderer.drawText(mFontBold, "Mars Type", mPlanets[1]->position() + NAS2D::Vector{64 - (mFontBold.width("Mars Type") / 2), -mFontBold.height() - 10}, NAS2D::Color::White);
-	renderer.drawText(mFontBold, "Ganymede Type", mPlanets[2]->position() + NAS2D::Vector{64 - (mFontBold.width("Ganymede Type") / 2), -mFontBold.height() - 10}, NAS2D::Color::White);
+	renderer.drawText(mFontBold, PlanetAttributes[0].name, mPlanets[0]->position() + NAS2D::Vector{64 - (mFontBold.width(PlanetAttributes[0].name) / 2), -mFontBold.height() - 10}, NAS2D::Color::White);
+	renderer.drawText(mFontBold, PlanetAttributes[1].name, mPlanets[1]->position() + NAS2D::Vector{64 - (mFontBold.width(PlanetAttributes[1].name) / 2), -mFontBold.height() - 10}, NAS2D::Color::White);
+	renderer.drawText(mFontBold, PlanetAttributes[2].name, mPlanets[2]->position() + NAS2D::Vector{64 - (mFontBold.width(PlanetAttributes[2].name) / 2), -mFontBold.height() - 10}, NAS2D::Color::White);
```

Comment:
With that change, the code becomes very symmetric, and could could further modify it to use a `for` loop.

Actually, if there was an easy way to match up entries between the `mPlanets` and `PlanetAttributes` collections, you could use a for-each loop.

Reply:
Agreed. I didn't design PlanetAttributes, I think Brett did. I just wanted to clean it up a bit to remove string literals out of the main code and into a single source that can later be improved upon.